pom.xml

```diff
@@ -5,11 +5,12 @@
         <artifactId>plugin</artifactId>
         <version>1.565.3</version>
     </parent>
-    <artifactId>bigpanda-jenkins</artifactId>
+
     <name>BigPanda Notifier</name>
+    <artifactId>bigpanda-jenkins</artifactId>
     <description>Notifies BigPanda of Jenkins builds</description>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/BigPanda+Jenkins+Plugin</url>
-    <version>1.2-SNAPSHOT</version>
+    <version>1.3.0</version>
```

Comment:
This is not how Maven works, non-SNAPSHOT versions are set during the release only

Reply:
Thank you for the comment! I will get this fixed